README.md

```diff
@@ -111,14 +109,12 @@ In vanilla JavaScript:
       const video = document.getElementById('my-video')
 
       // Set the credentials for the streaming
-      const yourStreamName = "..."
-      const yourStreamAccountId = "..."
+      const yourStreamName = '...'
+      const yourStreamAccountId = '...'
 
       // Define callback for generate new token
-      const tokenGenerator = () => millicast.Director.getSubscriber({
-        streamName: yourStreamName,
-        streamAccountId: yourStreamAccountId
-      })
+      const options: DirectorSubscriberOptions = { streamName, streamAccountId }
```

Comment:
I think this should be: 
`const options: DirectorSubscriberOptions = { streamName: yourStreamNAme, streamAccountId: yourAccountId }`